src/main/java/com/example/rebrain/model/UpdateCard.java

```diff
@@ -0,0 +1,9 @@
+package com.example.rebrain.model;
+
+import lombok.Data;
+
+@Data
+public class UpdateCard {
```

Comment:
It'd better to explicitly specify that this is DTO in the name. For instance `UpdateCardDto`